studio/src/locales/config.json

```diff
@@ -0,0 +1,17 @@
+{
+    "features":{
```

Comment:
Ideally these settings show come from the database based on the template selected